docker/Dockerfile.fastqc

```diff
@@ -0,0 +1,25 @@
+# FROM python:3.10.7-slim-bullseye
+FROM openjdk:11-jre-slim-bullseye
+# keep python instead of openjdk:11-jre-slim-bullseye, as the image above comes with all the required dependencies.
+
+# install default-jre as fastqc needs extra packages
+RUN apt-get update \
+  && apt-get install -y --no-install-recommends procps wget unzip default-jre perl pip \
+  && pip install awscli==1.22.50 \
+  && apt-get clean -y \
+  && rm -rf /var/lib/apt/lists/*
+
+ARG pathogen
+COPY docker/${pathogen}.deps /
+
+# install fastqc
+RUN . /${pathogen}.deps \
```

Comment:
How come you've gone with this way for installing all the deps? Couldn't we just use a package manager, or even copy their docker files?

Reply:
This is not too dissimilar to what we do for pangolin, where we copy the environment over, instead of fetching it with git clone. 

This does not install all the dependencies, but adds the variable versions to the docker image so that the code below can install the required dependency.

Reply:
I more mean is there not a more reliable, formally distributed means for getting this software than a wget and an install?
And the same goes for other images where we are just cloning git. If they are distributed via conda or as a docker image themselves, we shouldn't need to be cloning git and moving files around? It keeps it all a bit more robust...

Reply:
A few of those packages are not available via conda and the docker images were not found to be reliable. In other cases, the docker images were not available either. 
Conda is not reliable either, in fact many dependencies (or dependencies of dependencies) are not pinned and it can become incredibly hard to figure out what goes wrong when these issues occur. An issue like this occurred for ncov / ONT when artic updated muscle and broke the code.

For instance, fastqc does not come with dockerfile and the code owner does not release it on conda either.

Reply:
Also, in a few cases (e.g. pangolin), we need the latest versions, which are not necessarily readily available in docker containers.